app/controllers/collections_controller.rb

```diff
@@ -202,7 +231,7 @@ def delete_document_from_item
 
   def edit_collection
     begin
-      collection = validate_collection(params[:id], params[:api_key])
+      collection = validate_collection(sanitize_collection_name(params[:id]), params[:api_key])
```

Comment:
Every call that you make to `validate_collection` wraps the first parameter with `sanitize_collection_name`. If this is the first thing you have to do every time, it would make more sense to make this the first line in `validate_collection`. If that is then the only place that you call `sanitize_collection_name` you can remove the method and just replace is with its body (`name.downcase.delete(' ')`) since it is only one line and you would no longer gain anything by wrapping it in a method.

Also in every other case, `collection_name` is contained in `params[:collectionId]` but here it is contained `params[:id]`. is there any way to make this more consistent?